tests/playwright/src/model/pages/containers-page.ts

```diff
@@ -49,7 +49,21 @@ export class ContainersPage extends MainPage {
     return new ContainerDetailsPage(this.page, name);
   }
 
-  async stopContainer(container: string): Promise<ContainerDetailsPage> {
+  async startContainerFromContainersList(containerRow: Locator): Promise<ContainersPage> {
```

Comment:
I think that the function could be called simply `startContainer` and similarly for `stopContainer`, the context form where we call the function will make a distinction from details page. The same function will be in details page (not part of this PR). It will be then clear that we have something to refactor.

Reply:
Also, it would be better to pass string param as a container name. Not the locator.

Reply:
As @odockal says, please refactor the method to receive the container name, the locator identification should be done behind the scene.